[ZH] Update `Ranking_Criteria` (#6732)
ppy/osu-wiki: wiki/Ranking_Criteria/zh.md

# Ranking Criteria

*若想了解不同游戏模式具体的标准，参见 [osu!](osu!)，[osu!taiko](osu!taiko)，[osu!catch](osu!catch) 和 [osu!mania](osu!mania)。*

> **Ranking Criteria** 列出了让[谱面](/wiki/Beatmap)通过[谱面ranking程序](/wiki/Beatmap_ranking_procedure)时须遵守的[规则和准则](#一般术语)

Reviewer: 英文原文中存在`must`，为了强调遵守的重要性，建议保留原翻译中的词语"必须"

Author: 这里须就是必须的须，我只是让语序看起来更通顺一点

> 本文中规则和准则的改变会在 [Ranking Criteria 论坛](https://osu.ppy.sh/community/forums/87)中提出和讨论。参照 *[如何建议更改Ranking准则](https://osu.ppy.sh/community/forums/topics/720532)* 来提供贡献。注意，若在仅改变本文的语言、语法、组织，而不改变其原意的情况下则无需发帖至论坛。

Reviewer: ```suggestion
可在 [Ranking Criteria 论坛](https://osu.ppy.sh/community/forums/87)中提出和讨论对本文的更正。并参照 *[如何建议更改Ranking准则](https://osu.ppy.sh/community/forums/topics/720532)* 来提供贡献。注意，若仅对本文中的遣词、语法、组句进行改变，而不改变其原意，则无需发帖至论坛。
```
原文仅包含`this document`。故删去第一句的"规则和准则"。  
rules 在后文被翻译成“规定”，作为专有名词，应保持一致性。  
我认为原文翻译尚可，如果必须更改，可以调整为中文语序。  
`organisation` 的翻译我也拿不太准，个人推断原文的含义是"如何组织句子"。

> **注意[摸图和作图行为准则](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)、[八分拍歌曲测定](/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures)、[歌曲内容规定](/wiki/Rules/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有的游戏模式。**

Reviewer: ```suggestion
**注意[摸图和作图的行为准则](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)、[八分拍歌曲Timing测定](/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures)和[歌曲内容规定](/wiki/Rules/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有的游戏模式。**
```
"Timing"是一个在中文圈也较为常用的专业术语，可保留不翻。  
删去"和，的"个人认为会导致句子不通顺。

## 术语

### 一般术语

- **规定（Rules）：** 规定就是 **规定**。它们 **不是** 准则，在 **任何** 情况下都 **不得** 违反。
- **准则（Guidelines）：** 在**例外情况**下可以违反准则。这些例外情况必须能够详细解释，说明违反准则的原因，以及不违反准则将如何影响谱面的整体质量。

> ## 常规

Reviewer: 原翻译使用`一般`我个人觉得更好，如果想让本文更加书面化，不知可否写成`一般性原则`或`通用`？

Author: 我个人觉得一般太偏口语了，包括通用这个词。一般性原则太长了

### 规定

> - **两个物件不能出现在同一时间上。** 包括圆圈、滑条、转盘覆盖的全部时间段。osu!mania 模式除外。

Reviewer: 原文使用`tick`，建议保留原翻译"时间点"更好。 
如果觉得`circle`翻译成圈不正规，可考虑翻译成"单点"。这个需要和中文维基翻译团队确认标准术语翻译。

Author: 使用圆圈是可行的，不会让他看起来不正规。
因为之前红线、绿线有特定翻译“非继承时间点”，如果这里用时间点会造成混淆。

- **背景图片、背景视频、故事板不得出现任何违规内容。** 包括裸体、近乎裸体、性暗示、暴力、滥用药品（毒品）等元素。参见[视觉内容注意事项](/wiki/Rules/Visual_Content_Considerations)。

> - **必须在包含快速、重复的闪烁图像，或视频、故事板内含有对比度、颜色、亮度的快速变化内容的谱面中开启光敏性癫痫警告。** 如果警告会影响谱面游玩，则必须在歌曲前增加足够的空白时间。3 HZ（或：1秒变化3帧） 以下的闪烁图像一般无须担心。如果有疑问，请开启警告并在摸图时讨论是否有必要开启它。

Reviewer: ```suggestion
- **包含快速、重复的闪烁图像，或视频、故事板内含有对比度、颜色、亮度的快速变化内容的谱面，必须开启光敏性癫痫警告。** 如果警告会影响谱面游玩，则必须在歌曲前增加足够的空白时间。3 HZ（即：1秒闪烁3次） 以下的闪烁图像一般无须担心。如果有疑问，请开启警告并在摸图时讨论是否有必要开启它。
```
原文如下：  
> ... Strobing effects at 3 Hz (i.e. 3 flashes per second)  ...  

这里按照原文字面翻译即可，解释也可能造成歧义。    
另外，第一句话的脉冲图像可以考虑直接字面意思翻译，这里我拿不太准，等待其他reviewer的意见。

> - **谱面文件夹中不能有未使用文件或 0 字节的文件。** 0 字节的文件会影响文件夹中其他文件的正常上传。自动生成的 `thumbs.db` 文件除外。

Reviewer: ```suggestion
- **谱面文件夹中不能有未使用文件或 0 字节的文件。** 0 字节的文件会影响文件夹中其他文件的正常上传。仅自动生成的 `thumbs.db` 文件除外。
```

- **[休息时间](/wiki/Beatmap/Break)必须按照[谱面编辑器](/wiki/Client/Beatmap_editor)的限制插入。**

> - **如果相同游戏模式下的难度包含相同的休息时间或相同的故事板，则它们的`休息段使用黑边填充背景`设定必须一致，**

Reviewer: 这句话前面漏一条规定的翻译，原文如下：
> **Difficulty settings must not use more precision than is possible in the beatmap editor.**
可参考翻译如下：  
````
**难度的各项设定必须保证是在谱面编辑器手工设定时可复现的。** 
````
（这条规则主要限制诸如CS设个3.1415926这种可以改难度文件做到，但是在谱面编辑器中按住Shift设置也做不到的一些行为）

`Letterbox during breaks` 可以作为专有名词范畴，考虑到现在没有官方中文的谱面编辑器，这里建议保留原有名词，类似上面提到的这种专有名词保留问题全文还有很多，碍于时间关系这条review可能无法完整涉及，有时间我会继续摸完。

Author: 可能是翻译的时候漏掉了 感谢补充
而且 关于“专有名词”的翻译，我个人是感觉虽然守旧肯定是没问题的，但是已经过时而且有碍于玩家的交流。（现在玩家甚至简称为“黑边”）

Author: 已添加

> - **如果谱面包含剧透内容，则请在谱面描述中标注剧透警告。**

Reviewer: ```suggestion
- **如果谱面包含剧透内容，则必须在谱面描述中标注剧透警告。**
```
规定就是规定，使用"请"之类的环境更像是准则。

### 准则

> - **谱面使用的素材作者应列在谱面介绍内。** 参见[内容使用准则](/wiki/Rules/Content_Usage_Guidelines)，谱面使用的素材只有获得素材创作者的授权才能在 osu! 内使用。他们的授权可能会详细说明允许谱师如何合规使用这些素材。如果极难追溯到素材的来源、创作者允许无授权转载、授权已展示在艺术家、曲名、谱面元数据参考区域时，则通常无需在谱面介绍内再次标注。

Reviewer: ```suggestion
- **谱面使用素材的作者应列在谱面介绍内。** 参见[内容使用准则](/wiki/Rules/Content_Usage_Guidelines)，谱面使用的素材只有获得素材创作者的授权才能在 osu! 内使用。他们的授权可能会详细说明允许谱师如何合规使用这些素材。如果极难追溯到素材的来源、创作者允许无授权转载、授权已展示在艺术家、曲名、谱面元数据参考区域时，则通常无需在谱面介绍内再次标注。
```

> Creators of the media used in a beatmap should be credited in the beatmap's description. Per the Content Usage Guidelines, media used in beatmaps should be permitted for use in osu!, and their licensing will likely detail how to credit their creators. Common reasons to omit credits from the description include exceeding difficulty in sourcing, creators' wishes to remain uncredited, and presence of credits in the artist, title, or source fields.

这后面我看着是在说明license包含有哪些内容的样子，我不太确定，可能en版又更新了。  
介于时间关系这条摸先到这里，其余的我会慢慢看继续摸完。其实整体翻译的质量还是很不错的，我自己肯定很难做到啦。

## 音频

### 规定

- **谱面音频文件必须使用 `.mp3` 或者 `.ogg` 文件格式，且平均比特率不得不高于 192kbps。**
- **谱面只应包含一个用于所有难度的音频文件。** osu! 不支持单谱面集中的多个音乐文件，这会导致预览时间点、元数据等出现错误。
- **音频文件和音效必须有合理的质量。** 请尽可能尝试查找可用的最高质量音乐文件，而不是从流视频网站中提取。歌曲音量应被标准化为原始发布版本的音量，并且不应编码成高于原始文件的码率。
- **谱面必须添加音效。** 打击音效能给予玩家反馈，并且多样的音效（口哨、鼓掌和结束音）可以强调歌曲中最重要的部分。
  - **osu!mania 谱面可不添加打击音效。** 这可以使 osu!mania 谱师更轻松地制作谱面，当然，我们也推荐添加打击音效来润色谱面。如果不添加音效，则应注意非继承时间点的使用（除了调整段落默认打击音量）。
- **所有可击打的物件都必须包含能听见的[主动打击音效](/wiki/Beatmapping/Hitsound#active-hitsound)。** osu!mania 谱面除外，因为这个模式的节奏构成与其他模式不同。
- **[主动打击音效](/wiki/Beatmapping/Hitsound#active-hitsound)必须使用 `.wav` 或者 `.ogg` 格式。** `.mp3` 文件有轻微的延迟，因此只能在更长的[被动音效](/wiki/Beatmapping/Hitsound#passive-hitsound)中使用，比如环境噪音。
- **[主动打击音效](/wiki/Beatmapping/Hitsound#active-hitsound)的开头延迟时间不能超过 5 毫秒。** 「延迟」是从音效的峰值时间点向前计算，而不是音效的开始时间点计算。这保证敲击音效可以有效地给歌曲提供几乎同时的反馈。默认皮肤的 `normal-hitfinish.wav` 带有轻微的延迟，但仍可被用作自定义音效。
- **音效必须能让玩家感知到。** 它们的目的是提供反馈，所以音量极低的敲击音效或者由歌曲的原始样本合成的敲击音效都是不能使用的。具体游戏模式的例外情况请参阅它们相应的 Ranking Criteria。
- **同一谱面的预览时间点必须一致。** 这用于游戏中的歌曲选择菜单和网页中的在线歌曲试听。
- **每一个音效长度至少为 25 毫秒。** 长度更短的文件无法在游戏中播放出声音。
- **完全静音的文件必须使用[这个 44 字节文件](https://up.ppy.sh/files/blank.wav)。** 其他静音音效文件可能会过大，而 0 字节的文件不能使用。

> - **[故事板敲击音效](/wiki/Beatmapping/Hitsound#passive-hitsound)不能替换[主动打击音效](/wiki/Beatmapping/Hitsound#active-hitsound)。** 这会给玩家不准确的反馈。在其他情况下，可以但不鼓励使用故事板敲击音效。osu!mania 不受这一规定的约束。

Reviewer 2: ```suggestion
- **[故事板敲击音效](/wiki/Beatmapping/Hitsound#storyboarded-hitsound)不能替换[主动打击音效](/wiki/Beatmapping/Hitsound#active-hitsound)。** 这会给玩家不准确的反馈。在其他情况下，可以但不鼓励使用故事板敲击音效。osu!mania 不受这一规定的约束。
```

# Ranking Criteria

*查看每个游戏模式具体的标准，请参阅 [osu!](osu!)，[osu!taiko](osu!taiko)，[osu!catch](osu!catch) 和 [osu!mania](osu!mania)。*

> **Ranking Criteria** 列出了让[谱面](/wiki/Beatmap)通过[谱面ranking程序](/wiki/Beatmap_ranking_procedure)时须遵守的[规则和准则](#一般术语)

Reviewer 3: ```suggestion
**Ranking Criteria** 列出了让[谱面](/wiki/Beatmap)通过[谱面 Ranking 程序](/wiki/Beatmap_ranking_procedure)时须遵守的[规则和准则](#一般术语)
```

> 可在 [Ranking Criteria 论坛](https://osu.ppy.sh/community/forums/87)中提出和讨论对本文的更正。并参照 *[如何建议更改Ranking准则](https://osu.ppy.sh/community/forums/topics/720532)* 来提供贡献。注意，若仅对本文中的遣词、语法、组句进行改变，而不改变其原意，则无需发帖至论坛。

Reviewer 3: ```suggestion
可在 [Ranking Criteria 论坛](https://osu.ppy.sh/community/forums/87)中提出和讨论对本文的更正。并参照 *[如何建议更改 Ranking 准则](https://osu.ppy.sh/community/forums/topics/720532)* 来提供贡献。注意，若仅对本文中的遣词、语法、组句进行改变，而不改变其原意，则无需发帖至论坛。
```

> **注意[摸图和作图的行为准则](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)、[八分拍歌曲Timing测定](/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures)和[歌曲内容规定](/wiki/Rules/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有的游戏模式。**

Reviewer 3: ```suggestion
**注意[摸图和作图的行为准则](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)、[八分拍歌曲 Timing 测定](/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures)和[歌曲内容规定](/wiki/Rules/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有的游戏模式。**
```

## 术语

### 一般术语

- **规定（Rules）：** 规定就是 **规定**。它们 **不是** 准则，在 **任何** 情况下都 **不得** 违反。

> - **准则（Guidelines）：** 在**例外情况**下可以违反准则。这些例外情况必须能够详细解释，说明违反准则的原因，以及不违反准则将如何影响谱面的整体质量。

Reviewer 3: 规定使用了空格来表示强调 `规定就是 **规定**。`
但是准则这样做 统一一下？

## 常规

### 规定

- **两个物件不能出现在同一时间上。** 包括圆圈、滑条、转盘覆盖的全部时间段。osu!mania 模式除外。
- **背景图片、背景视频、故事板不得出现任何违规内容。** 包括裸体、近乎裸体、性暗示、暴力、滥用药品（毒品）等元素。参见[视觉内容注意事项](/wiki/Rules/Visual_Content_Considerations)。
- **包含快速、重复的闪烁图像，或视频、故事板内含有对比度、颜色、亮度的快速变化内容的谱面，必须开启光敏性癫痫警告。** 如果警告会影响谱面游玩，则必须在歌曲前增加足够的空白时间。3 HZ（即：1秒闪烁3次） 以下的闪烁图像一般无须担心。如果有疑问，请开启警告并在摸图时讨论是否有必要开启它。
- **谱面文件夹中不能有未使用文件或 0 字节的文件。** 0 字节的文件会影响文件夹中其他文件的正常上传。仅自动生成的 `thumbs.db` 文件除外。
- **[休息时间](/wiki/Beatmap/Break)必须按照[谱面编辑器](/wiki/Client/Beatmap_editor)的限制插入。**
- **[难度](/wiki/Client/Beatmap_editor/Song_Setup#difficulty)的各项设定参数必须保证能直接在[谱面编辑器](/wiki/Client/Beatmap_editor)内编辑复现。**
- **如果相同游戏模式下的难度包含相同的休息时间或相同的故事板，则它们的`休息段使用黑边填充背景`设定必须一致，**
- **如果谱面包含剧透内容，则必须在谱面描述中标注剧透警告。**

### 准则

> - **谱面使用素材的作者应列在谱面介绍内。** 参见[内容使用准则](/wiki/Rules/Content_Usage_Guidelines)，谱面使用的素材只有获得素材创作者的授权才能在 osu! 内使用。他们的授权可能会详细说明允许谱师如何合规使用这些素材。如果极难追溯到素材的来源、创作者允许无授权转载、授权已展示在艺术家、曲名、谱面元数据参考区域时，则通常无需在谱面介绍内再次标注。

Reviewer 3: ```suggestion
- **谱面使用素材的作者应列在谱面介绍内。** 参见[内容使用准则](/wiki/Rules/Content_Usage_Guidelines)，谱面使用的素材只有获得素材创作者的授权才能在 osu! 内使用。他们的授权可能会详细说明允许谱师如何合规使用这些素材。如果极难追溯到素材的来源、创作者允许无授权转载，或者授权已展示在艺术家、曲名、谱面元数据参考区域时，则通常无需在谱面介绍内再次标注。
```

Reviewer 3: 我超 那没事了

> 从描述中省略信用的常见原因包括寻找来源的难度过大、创作者希望保持无信用，以及艺术家、标题或来源字段中存在信用。

Reviewer 4: Credit 机翻？

Author: 参考了机翻 但是没改掉这个 改一改

- **不鼓励在别的 Ranked 谱面中重复使用你的 Ranked 难度。** 这是为了避免 Ranked 内容中不必要的重复。
- **不应通过编辑 `.osu` 文件来修改滑条点倍率。** 大多数自定义值会导致滑条点偏离于歌曲节奏，虽然有部分谱面使用了 0.5，1.333 和 1.5 这些值，但也只能在符合歌曲节奏时使用。
- **Kiai 时间应该从歌曲中一个明显的声音开始。** 否则，这会导致 kiai 闪光感觉上和歌曲不合拍。
- **相同游戏模式下所有难度的`启用倒计时`设置应该一致。** 如果某个难度前没有足够的开场时间而导致倒计时无法出现，则可忽略该准则。

## 谱面集

*注意：[osu!mania](/wiki/Game_mode/osu!mania)的各个键数在本小节都将视为一个单独的游戏模式。*

### 规定

- **谱面内的每个游戏模式，都必须从符合歌曲[掉血时间](/wiki/Gameplay/Drain_time)所允许的最低难度开始，到最高难度之间，有合理、连续的难度分布。** 不能跳过任何难度等级，任何相邻的两个难度之间也不能存在巨大的星数差距。

> - **每个难度必须遵循其模式的 ranking criteria 。** 参见 [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch)，和 [osu!mania](osu!mania) 的 ranking criteria 获取更多信息。

Reviewer 3: ```suggestion
- **每个难度必须遵循其模式的 Ranking Criteria 。** 参见 [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch)，和 [osu!mania](osu!mania) 的 Ranking Criteria 获取更多信息。
```

- **谱面最短的[掉血时间](/wiki/Gameplay/Drain_time) 为 30 秒。**
- **如果谱面的[掉血时间](/wiki/Gameplay/Drain_time) 符合以下条件：**
  - **小于 3:30，则最低难度不得高于 Normal。**
  - **在 3:30 到 4:15 之间，则最低难度不得高于 Hard。**
  - **在 4:15 到 5:00 之间，则最低难度不得高于 Insane。**
  - **最高难度以下的难度可以使用包含休息时间的总游戏时长来替代掉血时间，作为度量标准而达到规定阈值。** 这不包含掉血时间在 30 秒以下的难度。一般由该游戏模式的 [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) 和 [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) 成员来判断谱面中作为最低难度的 Hard 或者 Insane 谱面是否合格。

>   - **osu!mania 谱面则使用了不同的难度分布规则和最低难度限制。参见：[osu!mania ranking criteria](/wiki/Ranking_Criteria/osu!mania).**

Reviewer 3: ```suggestion
  - **osu!mania 谱面则使用了不同的难度分布规则和最低难度限制。参见：[osu!mania Ranking Criteria](/wiki/Ranking_Criteria/osu!mania).**
```

  - 注意：如果一个曲子同时被加速和剪切，则需使用组合标记 `(Sped Up & Cut Ver.)` 或 `(Nightcore & Cut Ver.)`.
- **特殊的 Unicode 字符必须使用与其最相似的标准等效字符，或者从 `.osu` 文件中的罗马音字段中删除。**`★ ☆ ⚝ ✪` 等替换为星号（`*`）。其他特殊字符则按情况使用或删除。
- **如果一个谱面集的音轨是由多首歌曲组合而成，将各曲名以清晰的分隔列出，或使用一个描述性的标题。** 如果分隔列出的曲名标题过长，则必须使用描述性的标题。
- **如果使用符号对标题的某些部分分组，则必须在分组之前和之后使用空格，但不能直接在分组中的符号之前或之后使用空格。**
- **如果一个作为来源字段系列含有子系列，则应使用最细化的子系列作为来源。** 如果歌曲有多个子来源可供使用，则应返回使用主来源字段。

#### 罗马音化

- **当谱师自行罗马音化处理艺术家名称，且没有官方罗马音化方案可供参考时，艺术家名称必须按照 Unicode 字段中出现的顺序进行罗马音化。**
- **其他语言的外来词在进行罗马音化处理时，必须使用借用的原词而并非罗马音化它们。**
- **当曲名或艺术家中使用表意重复的单词（其中一个是 Unicode，一个是罗马音）时，只能使用该罗马音作为罗马音化字段，并且删除对应的 Unicode 字符。**
- **带有德语元数据的歌曲必须将变元音罗马音化为两个字母的等效表示： `ü` 变为 `ue`，`ö` 变为 `oe`，`ä` 变为 `ae`，`ß` 变为 `ss`.**
- **带有俄语或西里尔字符的元数据必须使用 BGN/PCGN 体系来罗马音化。** 同样适用于 Source 字段，如果作者偏好罗马音化表达。如果 `Е` 和 `е` 单独出现，或在 `a`，`e`，`ё`，`и`，`о`，`у`，`ы`，`э`，`ю`，`я`，`й`，`ъ`，`ь` 之后出现，应该罗马音化为 `ye`。在其他情况中，应该罗马音化为 `e`。`ё` 应该罗马音化为 `ye`，但是，使用 `yo` 或 `o` 以避免使用特殊字符。若不使用提供的文件，而使用其他规则都是无关紧要，或是对游戏没有帮助的。对于大部分其他字符的转换规则，请参阅[这个文件的第一页](https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/807920/ROMANIZATION_OF_RUSSIAN.pdf)。

> - **带有日文元数据的歌曲必须使用[平文式罗马字](https://en.wikipedia.org/wiki/Hepburn_romanization#Features)罗马音化。** 如果作图者偏好罗马音的 Source，则也应用于该字段。对于非 Unicode 域，长元音如 `おう` 和 `うう` 应该写作罗马音 `ou` 和 `uu`，以避免长元音符号。

Reviewer 3: ```suggestion
- **带有日文元数据的歌曲必须使用[平文式罗马字](https://zh.wikipedia.org/wiki/%E5%B9%B3%E6%96%87%E5%BC%8F%E7%BD%97%E9%A9%AC%E5%AD%97)罗马音化。** 如果作图者偏好罗马音的 Source，则也应用于该字段。对于非 Unicode 域，长元音如 `おう` 和 `うう` 应该写作罗马音 `ou` 和 `uu`，以避免长元音符号。
```
再不济这个也行（
https://baike.baidu.com/item/%E5%B9%B3%E6%96%87%E5%BC%8F%E7%BD%97%E9%A9%AC%E5%AD%97/4513631

Author: ok
